gatkcwlgenerator/json2cwl.py

```diff
@@ -57,12 +57,17 @@ def json2cwl(GATK_json, cwl_dir, cmd_line_options):
     skeleton_cwl = {
         'id': GATK_json['name'],
         'cwlVersion': 'v1.0',
-        'baseCommand': ['java', '-jar', '/gatk/GenomeAnalysisTK.jar'],
+        'baseCommand': ['java', '-jar', cmd_line_options.gatk_location],
         'class': 'CommandLineTool',
+        'hints': [{
```

Comment:
This should be requirements as these descriptions on work for a particular Docker image